src/include/connection.h

```diff
@@ -335,6 +335,8 @@ struct __wt_connection_impl {
 	wt_thread_t	 sweep_tid;	/* Handle sweep thread */
 	int		 sweep_tid_set;	/* Handle sweep thread set */
 	WT_CONDVAR	*sweep_cond;	/* Handle sweep wait mutex */
+	time_t		 sweep_interval;/* Handle sweep interval */
+	time_t		 sweep_idle_time;/* Handle sweep idle time */
```

Comment:
Please swap their order to alphabetize.  I wouldn't mind if you alphabetized the entire group of sweep fields.